### ISSUE TITLE
Unable to edit the default help text of the Webmention form

### ISSUE THREAD

**Issue by jeremycherfas** [pfefferle/wordpress-webmention#177]
I clicked on `Change the default help text of the Webmention form` and that highlighted the text box and placed the cursor at the beginning of the text, but I was not able to make any changes.

**Comment by pfefferle** [comment]
You can‘t type?

**Comment by pfefferle** [comment]
FYI the default text is only the `placeholder` text, you can not edit it, but add your own version.

**Comment by jeremycherfas** [comment]
I expected to be able to edit it, rather than have to supply a complete
new text of my own. Lazy I suppose. And as soon as I start to type, the
placeholder vanishes completely. So, just unexpected and not very
helpful, I suppose.

On Mon, 21 May 2018, at 1:01 PM, Matthias Pfefferle wrote:
> FYI the default text is only the placeholder text, you can not edit
> it, but add your own version.> — You are receiving this because you authored the thread. Reply to
> this email directly, view it on GitHub[1], or mute the thread[2].

Links:

  1. https://github.com/pfefferle/wordpress-webmention/issues/177#issuecomment-390623086

**Comment by dshanske** [comment]
Maybe the default should appear above it.

**Comment by jeremycherfas** [comment]
I think a clearer message is all that is needed. Something like "Replace the placeholder text above. (You may want to copy it to the clipboard before you start."

**Comment by tw2113** [comment]
I agree with Jeremy here. The amount of text in the placeholder is way too much and the overall area is not as clear as it could or should be. Chances are people aren't going to want to replace the entirety of the message, but instead touch up words here and there.

I could see:

1. providing the default nearby, either above or below the field that they could then copy/paste.
	1. Con: making user copy/paste text. Could offer a button to easily copy the text though, like how GitHub offers commit hash easy copying.
2. providing the default as editable text in the textarea right away.
